scripts/kind-e2e/config/cluster1-config.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Cluster
```

Comment:
Wouldn't it be less maintenance burden if you clone submariner somewhere and use those files from there?, you'd need to maintain only once.

Reply:
Yes you are correct, we already spoke about internally. The idea is to create a tool based on kind that will just accept parameters and all the projects will use it. This will also remove the dependency on dapper. Some suggestions for names are armada and [futtock](https://www.google.com/search?q=futtock&oq=futtock&aqs=chrome..69i57j69i61.1933j0j7&sourceid=chrome&ie=UTF-8)  
:)

Reply:
The problem is that if we make changes to submariner e2e and break its e2e we are braking all the projects together.

Reply:
Well, but shouldn't CI prevent such breakage in first instance? :) I'd be fine with that.

Reply:
Why another tool?, 

Remember that we need to maintain those tools, and compile for other platforms, for example, I use dapper in OSX....

Reply:
Yes you are right but there could be always an edge case, its lame argument :) i know, another point is that each project has its own custom flow, so pulling all submariner stuff is not needed and each project can have its own upgrade path and validate and test logic.

Reply:
The issue is that we’re sharing a bunch of code, which means we need to revisit the situation (and one possible conclusion is that it’s fine where it is, but we do need to think about it). If we pull the e2e code out as a separate project, then we’d have project management on it, _e.g._ releases, so other projects could depend on a version they know works, and we wouldn’t force unintentional breakage on them...

Reply:
Yes to have  a smart provisioner in separate repository and all the projects could utilize it is seems the way to go.